docs/how-pipx-works.md

```diff
@@ -16,6 +16,8 @@ When installing a package and its binaries on linux (`pipx install package`) pip
 - as long as `~/.local/bin/` is on your PATH, you can now invoke the new binaries globally
 - on operating systems which have the `man` command, as long as `~/.local/share/man` is a recognized search path of man,
   you can now view the new manual pages globally
+- adding `--global` flag to any `pipx` command will execute the action in global scope which will expose app to all
```

Comment:
Please also mention here that this is not available on windows.

Reply:
Done